src/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title><%= htmlWebpackPlugin.options.title %></title>
+  </head>
+  <body>
+    <div class="container">
+      <!-- Header -->
+      <header>
+        <a href="">
+          <img
+            src="https://placehold.co/40x40/orange/white?font=Lora&text=YAS"
+            alt=""
+          >
+          <span>Your Amazing Shows</span>
+        </a>
```

Comment:
- Kindly, you missed showing the number of movies :+1: so please add a count function that counts the number of movies and displays it, and doesn't use API response.length to show the movie count instead count the card list because you will use this function later for  testing :+1: 
![Screenshot from 2023-03-18 20-54-32](https://user-images.githubusercontent.com/52010789/226126046-65dc86e3-2230-4de3-bb10-04ebc17a8b8b.png)
![Screenshot from 2023-03-17 22-40-07](https://user-images.githubusercontent.com/52010789/226126061-72f6d88f-d02f-47be-91c2-f3fc8f908adc.png)